docs/fundamentals/custom-resource-urls.md

```diff
@@ -0,0 +1,79 @@
+---
+title: Custom resource URLs
+description: Learn how to create custom URLs for .NET Aspire resources.
+ms.date: 04/08/2025
+ms.topic: how-to
+---
+
+# Custom resource URLs
+
+.NET Aspire resources that expose endpoints only configure host and port values. However, there might be situations where you want to access a specific route of an exposed endpoint. The host and port are unknown until run time. In these cases, you can use custom resource URLs to define specific routes on a configured endpoint, which is convenient for accessing resources from the [dashboard](dashboard/overview.md).
```

Comment:
Not a fan of using the term "route" here as I associate that more with the features in app frameworks to define patterns for incoming request matching. Can we rephrase this to not use the term "route" and instead use terms common with URLs, e.g. "path"?

Reply:
Yeah, I agree - "path" is better and more common terminology. I'll fix that.